tests/tasks/test_datachecks.py

```diff
@@ -0,0 +1,165 @@
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for the datacheck tasks."""
+
+import datetime
+
+import tasksmodule
+import utils
+
+import task_tests_base
+
+
+class PersonDataValidityCheckTaskTests(task_tests_base.TaskTestsBase):
+    """Tests the person record validity datacheck task.
+
+    We don't have a unit test for the missing entry date case, because
+    entry_date is required by the model (i.e., we cannot test it because we
+    cannot create a Person entity without it to test with).
+    """
+
+    def init_testbed_stubs(self):
+        self.testbed.init_user_stub()
```

Comment:
just by habit, should we set testbed.tear_down too?

Reply:
`TaskTestsBase` does that already. This function just exists here because these tests require more stubs than some other tests.